src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -8019,12 +8046,15 @@ namespace SurfaceGeometry {
 					kivaManager.settings.deepGroundBoundary = HeatBalanceKivaManager::KivaManager::Settings::ZERO_FLUX;
 				} else if (SameString(cAlphaArgs( alpF ), "GroundWater")) {
 					kivaManager.settings.deepGroundBoundary = HeatBalanceKivaManager::KivaManager::Settings::GROUNDWATER;
-				} else /* if (SameString(cAlphaArgs( alpF ), "Autoselect")) */ {
+				} else if (SameString(cAlphaArgs( alpF ), "Autoselect")) {
```

Comment:
@nealkruis I'm fine with Autoselect, because it's choosing one of the other choices, not calculating something.  But now that the code is enforcing what always was in the IDD, transition needs to trap any Autocalculates and convert them to Autoselect.  I transitioned ZoneCoupledKivaWalkoutBasement from v8.7 to v8.8 and it fails, because it's still got Autocalculate here.